docs/notebooks/interfaces/fetcher.ipynb

```diff
@@ -27,8 +27,8 @@
    "source": [
     "import obsplus\n",
     "\n",
-    "ta_dataset = obsplus.load_dataset('ta')\n",
-    "crandall = obsplus.load_dataset('crandall')\n",
+    "ta_dataset = obsplus.load_dataset('ta_test')\n",
```

Comment:
> reasonably sized

It currently has to load the whole event dataframe. This is fine if the reference is "origin" because it just pulls it from the bank. If it is related to pick times, however, it currently reads all the events into memory, extracts pick times, then begins the waveform extraction. I guess it could be done with generators in batches though, I will look more into this.

Reply:
> infinite loop

Yes there was a bug in the argument order. I fixed it.